packages/common/src/HardhatConfig.js

```diff
@@ -30,7 +31,10 @@ function getHardhatConfig(configOverrides, workingDir = "./", includeTruffle = t
 
   const defaultConfig = {
     solidity: {
-      compilers: [{ version: solcVersion, settings: { optimizer: { enabled: true, runs: 1000000 } } }],
+      compilers: [
+        { version: solcVersion, settings: { optimizer: { enabled: true, runs: 1000000 } } },
+        { version: "0.7.6", settings: { optimizer: { enabled: true, runs: 1000000 } } },
```

Comment:
Do you need to set 0.7.6 in `compilers`? I thought you only had to set it in the `ovm` value. [docs](https://community.optimism.io/docs/resources/hardhat.html#migrate-the-sample-app-to-optimistic-ethereum)

Reply:
yup _but_ when writing L2 tests (not e2e tests) we will run them in the normal HRE so we need our normal hardhat compiler to _also_ support this version to run in the HRE.

Reply:
aha